types/bls_to_execution_change.yaml

```diff
@@ -0,0 +1,23 @@
+BLSToExecutionChange:
+  type: object
+  description: "The [`BLSToExecutionChange`](https://github.com/ethereum/consensus-specs/blob/master/specs/capella/beacon-chain.md#blstoexecutionchange) object from the CL spec."
+  properties:
+    validator_index:
+      $ref: "./primitive.yaml#/Uint64"
+      description: "Index of the validator for which credentials will be changed."
+    from_bls_pubkey:
+      $ref: "./primitive.yaml#/Pubkey"
+      description: "Public key of existing credentials."
+    to_execution_address:
+      $ref: "./primitive.yaml#/ExecutionAddress"
+      description: "Execution address to which the credentials will be changed."
+
+SignedBLSToExecutionChange:
+  type: object
+  description: "The [`SignedBLSToExecutionChange`](https://github.com/ethereum/consensus-specs/blob/master/specs/capella/beacon-chain.md#signedblstoexecutionchange) object from the CL spec."
```

Comment:
might be worth mentioning this is signed with the Withdrawal key, rather than the validator key?

Reply:
Shouldn't that be in the spec rather than in the API documentation?  We're just following the spec, after all.

Reply:
sure, just wondering if its useful context to someone trying to consume the api doc

Reply:
Unsure, to be honest.  We don't specify this type of information anywhere else, but then this is somewhat an exception when compared to the other validator signatures.

I'm coming down on the side of having it just in the spec, because as a rule I don't like to have things defined in multiple places.  But it's not a strong preference.

Reply:
yep that's fair, if we get repeated issues we can clarify...